testsuite/settings.py

```diff
@@ -4,4 +4,4 @@
 os.environ['SECRET_KEY'] = 'sheap'
 os.environ['LIQUID_2FA'] = 'true'
 
-from liquidcore.site.settings import *
+from liquidcore.site.settings import *  # noqa: F401, F403
```

Comment:
Not really sure how to get rid of F401 and F403.

Reply:
Read the docs: http://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html#in-line-ignoring-errors

In this specific case you could even ignore all errors for this line: `# noqa`